### ISSUE TITLE
deprecated usage of `--format` option

### ISSUE THREAD

**Issue by depressiveRobot** [samueltbrown/gradle-cucumber-plugin#45]
When executing `gradle cucumber` the following warning is printed:

```
WARNING: Cucumber-JVM's --format option is deprecated. Please use --plugin instead.
```

**Comment by pbellchambers** [comment]
There's a pull request which resolves this, however it's not been merged yet: #41

**Comment by viphe** [comment]
Merged and targeted at v0.9.

Deep apologies for the huge delays.
